src/osd/ReplicatedBackend.cc

```diff
@@ -1216,9 +1210,9 @@ void ReplicatedBackend::sub_op_modify_impl(OpRequestRef op)
   rm->localt.register_on_applied(
     parent->bless_context(
       new C_OSD_RepModifyApply(this, rm)));
-  list<ObjectStore::Transaction*> tls;
-  tls.push_back(&(rm->localt));
-  tls.push_back(&(rm->opt));
+  vector<ObjectStore::Transaction> tls;
```

Comment:
I have another pull request pending that is removing two transaction object from replication path same thing we did for submit_transaction()..
https://github.com/ceph/ceph/pull/7278
If it is correct, we don't need this..
But, probably as an independent pull request, I should do that (?)